include/OpenSpaceToolkit/Astrodynamics/Flight/System/PropulsionSystem.hpp

```diff
@@ -22,34 +18,12 @@ namespace system
 
 using ostk::core::type::Real;
 
-using ostk::physics::data::Direction;
-using ostk::physics::data::Scalar;
-using ostk::physics::unit::Length;
 using ostk::physics::unit::Mass;
-using ostk::physics::unit::Time;
-using ostk::physics::unit::ElectricCurrent;
-using ostk::physics::unit::Angle;
-using ostk::physics::unit::Derived;
-using ostk::physics::Unit;
 
 /// @brief Define a propulsion system (constant thrust, constant Isp for now)
 class PropulsionSystem
 {
    public:
-    static Unit thrustSIUnit;
-    static Unit specificImpulseSIUnit;
-    static Unit massFlowRateSIUnit;  // TBI: Define in ostk physics as proper units
-
-    /// @brief Constructor
-    ///
-    /// @code{.cpp}
-    ///              PropulsionSystem propulsion = { ... };
-    /// @endcode
-    ///
-    /// @param aThrust Thrust (scalar)
-    /// @param aSpecificImpulse Specific impulse (scalar)
-    PropulsionSystem(const Scalar& aThrust, const Scalar& aSpecificImpulse);
-
```

Comment:
Even though I am removing this constructor, its not really a breaking change because we don't use it anywhere downstream in cpp (except for the `PropulsionSystem.test.cpp` that I also refactored in this MR, and it doesn't exist in Python.